Mathlib/Util/Superscript.lean

```diff
@@ -263,6 +317,11 @@ def superscriptTerm := leading_parser (withAnonymousAntiquot := false) superscri
 
 initialize register_parser_alias superscript
 
+open PrettyPrinter.Delaborator in
+/-- Checks that the provided expression can be superscripted before delaborating. -/
+def delabSuperscript (e : Expr) : Delab :=
+  Superscript.superscriptable e >>= fun () ↦ delab
```

Comment:
Should this be using the "current" expression, rather than taking an unrelated expression as an argument?

Reply:
I think you're correct here, I just didn't know it was possible to extract the current expression.

Reply:
I would believe you were it not for the fact that you wrote `(← getExpr)` below!